.github/workflows/build_esp32.yml

```diff
@@ -40,6 +40,7 @@ jobs:
         - 'targett_mcb_wrover'
         - 'unexpectedmaker_feathers2'
         - 'unexpectedmaker_tinys2'
+        - 'unexpectedmaker_feathers2_neo'
```

Comment:
ABC order, should move 1 line up

Reply:
Sorry, I didn't realise it was alphabetical beyond the starting word.